src/utils/editor/editor-engtokor.util.ts

```
@@ -21,6 +21,8 @@ export const convertEngToKor = (type: string | null) => {
       return CATEGORY.SOCIAL;
     case ElEMENT_TYPE.TEMPLATE:
       return CATEGORY.TEMPLATE;
+    case ElEMENT_TYPE.TEMPLATE:
+      return CATEGORY.TEMPLATE;
```

Biome (1.9.4)

[error] Duplicate case label.

The first similar label is here:


(lint/suspicious/noDuplicateCase)